Posts/2021/09/tfl-profile.md

```diff
@@ -0,0 +1,126 @@
+---
+post_title: How to Make Use Of PowerShell Profile Files
+username: tfl@psp.co.uk
+Categories: PowerShell
+tags: PowerShell, profile
+Summary: Using Profile files with PowerShell 7
+---
+
+**Q:** I would like to personalize the way that PowerShell works. 
+I have heard that I can use a thing called a profile to do this, but when I try to find information about profiles, I come up blank. There is no ``New-Profile`` cmdlet, so I do not see how to create such a thing. Can you help me, please?
+
+**A:** Profile files are a powerful part of PowerShell and allow you to customize PowerShell for your environment.
```

Comment:
"Profile files" is kind of redundant. The **Profile** is the concept we want to describe. The file is an implementation detail.

```suggestion
**A:** Profiles are a powerful part of PowerShell and allow you to customize PowerShell for your environment.
```